### PR TITLE
Ran pimpmychangelog

### PR DESCRIPTION
**Pull request by pcreux**
👋 I just ran [`pimpmychangelog`](https://github.com/pcreux/pimpmychangelog) to pimp your CHANGELOG. It adds links to pull requests and users making it easier to jump to a specific PR to get more details.

### CODE REVIEW

**Comment by marcotc** [comment]
👋 @pcreux, thank you very much for this contribution!

I think the changelog is much easier to navigate on GitHub after this change.

One question I have is about the maintenance of the section below ` <!--- The following link definition list is generated by PimpMyChangelog --->`.

If during our next release we add this line to our changelog:
```md
- Pimp the changelog (#1267) (@pcreux)
```

Will `pimpmychangelog` automatically add the missing links to that bottom section (I assume at a predictable order)? If so, we'd likely need a Rake task that allows us to easy run the tool in the repo.

In other words, do we ever have to manually touch that section?

**Comment by pcreux** [comment]
Hello @marcotc !

Thank you for reviewing my PR.

Yes, `pimpmychangelog` will automatically add the missing links when you run it. Here is [an example](https://github.com/pcreux/csv-importer/commit/3e4b9c35531113778ad6dab041a2dec849645765#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed).

I let you add a Rake task, pre-commit hook and/or github action to run it.

**Review by marcotc — APPROVED** [review]
